cpp/ps0002/problem 2 p301F Salwa مساعدة بسيطة.cpp

```diff
@@ -4,25 +4,29 @@ using namespace std;
 
 int main()
 {
-//مساعدة بسيطة
-
-Short Int Num_residents = 30;
-
-Short Int medicine-alternative= 30;
-
-str Pr_members= {“Ahmed, Mohamed, Kareem, Ali”};
-
-float price = 25.5f;
-
-Char First_let= ‘A’;
-
-Long long int Num_babies = 1000,000,000;
-
-float result = 25.5f;
-
-double electricity = 1000,000,000;
-
-Long long int Age = 1000,000,000;
-
+    //مساعدة بسيطة
+    
+    short int Num_residents = 30; // 0.75/0.75
+    
+    short int medicine_alternative = 30; // 0.5/0.75
+    
+    short Pr_members = 10; // 0/0.75
+    // string names[] = {"Ahmed", "Mohamed", "Kareem", "Ali"}; // if u want names
+    
+    float price = 25.5f; // 0.75/0.75
+    
+    char First_let = 'A'; // 0.75/0.75
+    
+    int Num_babies = 1'000'000; // 0.25/0.75
+    
+    double result = 25.54367743; // 0.25/0.75
+    
+    double electricity = 1'000'000'000; // 0.75/0.75
+    
+    unsigned int Age = 1'000'000'000; // 0.5/0.75
+    
+    // total 4.5/6.75
+    
     return 0;
 }
+// good work 👍 7.5/10

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [x] sorted the declarations of variables or commented some phrases for blocks of code